src/test/java/task24/WebDriverCalculatorTestFour.java

```diff
@@ -0,0 +1,74 @@
+package task24;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import task24.calculator.page.CloudCalculatorPage;
+import task24.calculator.page.EstimateWindowPage;
+import task24.calculator.page.StartCloudPage;
+import task24.calculator.page.TenMinutesMailPage;
+import java.awt.datatransfer.UnsupportedFlavorException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WebDriverCalculatorTestFour {
+    private final String MAIL_PAGE_URL = "https://10minutemail.com/";
+    private WebDriver driver;
+    private int webDriverTimeOut = 10;
+
+    @BeforeMethod(alwaysRun = true)
+    public void browserSetUp() {
+        driver = new ChromeDriver();
+    }
+
+    @Test(description = "Test N4 - Send Email and check of total cost")
+    public void completeGoogleCalculatorAndSendEmail() throws IOException, UnsupportedFlavorException {
+
+        WebDriverWait webDriverWait = new WebDriverWait(driver, webDriverTimeOut);
+        JavascriptExecutor executor = (JavascriptExecutor) driver;
```

Comment:
JavascriptExecutor doesn't relate to the test. It's better to implement it in an exact screen or in a base abstract class.